index.html

```diff
@@ -2,10 +2,128 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <link rel="stylesheet" href="styles/styles.css">
-    <title>Title</title>
+    <link rel="stylesheet" href="styles.css">
+    <title>Concept opdracht</title>
 </head>
+
 <body>
+    <header class="navbar-outer-container">
+    <div>
+        <ul class="navbar-items-inner-container">
+            <li><a href="#case-studies" style="text-decoration:none ">Case Studies</a></li>
+            <li><a href="#services" style="text-decoration:none">Services</a></li>
+            <li><a href="#contact-us" style="text-decoration:none">Contact Us</a></li>
+        </ul>
+    </div>
+    <div class="intro-outer-container">
+        <h1 class="intro-h1-inner-container">Your website is an investment in the success of your business.</h1>
```

Comment:
Goed dat je je classes een beschrijvende naam geeft. Maar deze titel is geen inner-container, toch? Er staan verder geen items meer in dit element. Dus into-title is wellicht een betere class-naam!